documentation/ipi-install/modules/ipi-install-diagnosing-duplicate-mac-address.adoc

```diff
@@ -1,47 +1,53 @@
 [id="ipi-install-diagnosing-duplicate-mac-address_{context}"]
 = Diagnosing a duplicate MAC address when provisioning a new host in the cluster
 
-[role="_abstract"]
```

Comment:
Was the intention to remove the role too?

Reply:
Yes - got some feedback that altough CCS require this for new content, OpenShift team don't like it...